apiserver/facades/agent/uniter/uniter_test.go

```diff
@@ -629,7 +629,7 @@ func (s *uniterSuite) TestNetworkInfoSpaceless(c *gc.C) {
 
 	args := params.NetworkInfoParams{
 		Unit:      s.wordpressUnit.Tag().String(),
-		Endpoints: []string{"db"},
+		Endpoints: []string{"db", "juju-info"},
```

Comment:
This is a red/green test for the change. I'm surprised we had nothing like this in tests.